Source/Android/app/src/main/res/values/arrays.xml

```diff
@@ -433,6 +435,19 @@
         <item>Right Stick</item>
     </string-array>
 
+    <string-array name="drumsKit">
+        <item>-</item>
+        <item>+</item>
+        <item>Green</item>
+        <item>Red</item>
+        <item>Blue</item>
+        <item>Yellow</item>
+        <item>Orange</item>
+        <item>Bass 1</item>
+        <item>Bass 2</item>
```

Comment:
I'm wondering whether having two bass drums in the overlay actually makes sense.

I'm assuming that the reason you made it this way is because it's possible to use two bass pedals with a real drum controller. However, the gameplay point of connecting two bass pedals is to gain the ability to play the bass drum more quickly by alternating between two limbs – something which is already possible for the 3 pads and 2 cymbals without any extra hardware (you just need to use both arms at once on the same pad). The latter is far more important for gameplay, since the game only expects you to do double bass if you're playing on Expert+, yet this pull request is more accommodating towards playing double bass than playing fast rolls on the pads/cymbals.

Additionally, I (at least as a Rock Band player – I must admit I'm not entirely familiar with how it works in Guitar Hero drum hardware) think it's natural for the user to expect that if they hit the first bass drum and then hit the second bass drum before having released the first one, the hit on the second bass drum registers. As I understand it, this is not the case in this implementation, since you have simply mapped both pads to the same input.

Reply:
the purpose of inserting two pedals is all about positioning. at least if the user needs it, then the default value is false, and it only works in Overlay and not for external gamepads.

Reply:
Ah, so it's a matter of having a conveniently placed bass drum that you can hit no matter where your hand is? Hmm... I suppose that makes sense in a way.

Reply:
> 
> 
> Ah, so it's a matter of having a conveniently placed bass drum that you can hit no matter where your hand is? Hmm... I suppose that makes sense in a way.

Wouldn't a wide button that can be reached by either finger make more sense then?

Reply:
I think I agree.

Reply:
umm if we make the bass button wider like space button in real keyboard, the only way i can do is editing in the InputOverlayDrawableButton.java resizing before draw, since all image bound is treated as square, without resizing if image has empty space that will cause button got pressed when we tap its empty area, like the classic button L & R

Reply:
I have submitted PR #9944 to fix this issue.